src/handlers/localstorage/accountLocal.ts

```diff
@@ -64,14 +68,7 @@ export const saveSavings = (savings: any, accountAddress: any, network: any) =>
  * @return {Boolean}
  */
 export const getAccountEmptyState = (accountAddress: any, network: any) =>
-  getAccountLocal(
```

Comment:
hey i like ur thinking - in terms of the data itself: for this one since it's just a simple helper flag, we can just "dump" it and start fresh and not worry about a migration

separately, since we're "deleting" a flag, we should have a migration for deleting old keys (there are other keys we should also delete, we'll need to scrape which ones we've removed recently)